src/util/fees.cpp

```diff
@@ -26,16 +29,32 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
+const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap() {
+    static const std::vector<std::pair<std::string, FeeEstimateMode>> fee_modes = {
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
     };
-    auto mode = fee_modes.find(mode_string);
+    return fee_modes;
+}
 
-    if (mode == fee_modes.end()) return false;
+std::string FeeModes(const std::string& delimiter) {
+    std::ostringstream r;
```

Comment:
457396a4091bb3c27b1e982fb7e08442120b15df

Is it really necessary to `#include <sstream>` just to concat a couple of strings? Could just do
```cpp
for (i : ...) {
    if (!res.empty()) res += delimiter;
    res += i.first
}
```

Reply:
I don't really see the harm in including it (it won't make the binary bigger or anything) but it's such a trivial set of operations that I doubt it'll matter in the long run.

I.e. removing the include and just stacking onto an `std::string`.